biome.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+    "formatter": {
+        "indentStyle": "space",
+        "indentWidth": 4
+    }
+}
```

Comment:
we could also leave it out, then all JSON files get formatted with tabs

Reply:
Let's keep it, too bad biome doesn't support editorconfig.

Reply:
It’s coming, actually! https://github.com/biomejs/biome/issues/1724